CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Changelog
 ### next
+  * tpm2_activatecredential: -f becomes -I
```

Comment:
One of the goals for 4.0 is consistent naming of options as best as possible, focusing on tools in the same group class. I'd like to see capital letters for parent object arguments and lowercase for object arguments.

Well this does address an inconsistency it seems, can you perhaps propose something else?

Reply:
We could name the short option `-i` of course, which would follow the parent/object logic better. In this case, this should be done in `tpm2_create` and `tpm2_rsadecrypt` as well. The only remaining tool to use `--input-file` is `tpm2_encryptdecrypt`, but that one already has a short option `-i` for the initialisation vector `--iv`. What do you think?

Reply:
@diabonas seems reasonable.